astropy/table/table.py

```diff
@@ -2743,6 +2744,30 @@ def from_pandas(cls, dataframe):
 
         return cls(out)
 
+    def get_pos(self, name):
+        """
+        Get the position(index) of the specified column ``name`` in the Table.
+        
+        Parameters
+        ----------
+        name : str
+            Column name
+
+        Returns
+        -------
+        position : int
+            Position(index) of the specified column name
+        """
+
+        if self.positions is None:
+            self.positions = OrderedDict()
+            index = 1
+            for colname in self.columns:
+                self.positions[colname] = index
+                index += 1
+        
+        return self.positions[name]
+
```

Comment:
This does not need it's own function. Instead of `x = self.get_pos(name)` you can just do:
```
x = self.colnames.index(name)
```